web/modules/custom/dbcdk_community/src/Plugin/Block/ProfileBlock.php

```diff
@@ -173,6 +174,21 @@ protected function parseProfile(Profile $profile, array $fields) {
           ];
           break;
 
+        // The profile doesn't contain its roles so we cannot get the value as
+        // a regular property so we have to make a separate request for roles.
+        // The display of this field is also different than a single string
+        // value since we would like the roles to be presented as a list to make
+        // it easier overview.
```

Comment:
Appreciate the level of commenting here. Could the value be as in description to avoid the `render()` call?

 ```php
$value = [
    'data' => [
        '#theme' => 'item_list',
        '#list_type' => 'ul',
        '#empty' => $this->t('This user has not roles.'),
        '#items' => $this->getProfileRoles($profile),
    ]
]
```